db-helpers.js

```diff
@@ -103,24 +105,11 @@ function getUsers (callback) {
     })
 }
 
-function getUserByEmail (email, callback) {
+function getFormSubmission (email, dateSubmit, callback) {
     openDbConnection( (err, client, done) => {
-        client.query('SELECT * FROM users WHERE email = $1', [email], function (err, result) {
```

Comment:
It's because this PR changes the unique constraint on the table to the combination of email and date together. So a row isn't a user, it's more like an instance of a user submitting the form at a specific time. They could fill out the form more than once and their email could be in 2+ rows.

The thinking behind this change was that users might complete the form more than once. Maybe they fill it out once, decide not to join, and then weeks or months later change their mind and fill out the form for a second time.